src/frontend/magnify/src/components/design-system/CalendarInput/CalendarInput.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { IntlProvider } from 'react-intl';
-import { render, screen } from '@testing-library/react';
+import { act, render, screen, waitFor } from '@testing-library/react';
```

Comment:
Those variables are not used.
```suggestion
import { render, screen } from '@testing-library/react';
```